application/dash_application/views/call_ready_wgs.py

```diff
@@ -0,0 +1,549 @@
+from collections import defaultdict
+import logging
+import json
+
+import dash_core_components as core
+import dash_html_components as html
+from dash.dependencies import Input, Output, State
+import pandas as pd
+
+import gsiqcetl.column
+import pinery
+from ..dash_id import init_ids
+
+from ..utility.plot_builder import *
+from ..utility.table_builder import table_tabs, cutoff_table_data_merged
+from ..utility import df_manipulation as util
+from ..utility import sidebar_utils, log_utils
+
+logger = logging.getLogger(__name__)
+
+""" Set up elements needed for page """
+page_name = "call-ready-wgs"
+title = "Call-Ready WGS"
+
+ids = init_ids([
+		# Buttons
+		"update-button",
+
+		# Sidebar controls
+		"all-library-designs",
+		"library-designs-list",
+		"all-projects",
+		"projects-list",
+        "all-institutes",
+        "institutes-list",
+        "all-tissue-preps",
+        "tissue-preps-list",
+        "all-sample-types",
+        "sample-types-list",
+        "first-sort",
+        "second-sort",
+        "colour-by",
+        "shape-by",
+        "search-sample",
+        "show-data-labels",
+        "show-all-data-labels",
+        "cutoff-pf-tumour",
+        "cutoff-pf-normal",
+        "cutoff-coverage-tumour",
+        "cutoff-coverage-normal",
+        "cutoff-callability",
+        "cutoff-mean-insert",
+        "cutoff-duplicate-rate",
+
+		# Graphs
+		"total-reads",
+		"unique-reads",
+		"mean-coverage",
+		"callability",
+		"mean-insert",
+		"duplicate-rate",
+		"purity",
+		"ploidy",
+		"unmapped-reads",
+
+		# Tables
+		"failed-samples",
+		"data-table",
+])
+
+BAMQC_COL = gsiqcetl.column.BamQc3MergedColumn
+ICHOR_COL = gsiqcetl.column.IchorCnaMergedColumn
+CALL_COL = gsiqcetl.column.MutetctCallabilityColumn
+PINERY_COL = pinery.column.SampleProvenanceColumn
+
+
+def dataversion():
+    return DATAVERSION
```

Comment:
Any reason we couldn't just use DATAVERSION wherever this is called?

Reply:
I'd have to test that out further to see if it's set when the known_pages_router requests it... perhaps in a subsequent PR?

Reply:
Sure!